content/1_docs/2_cookbook/9_setup/0_multisite-variant/recipe.txt

```diff
@@ -0,0 +1,184 @@
+
+Even though the proposed setup is easy to accomplish, I dislike the fact of running multiple websites from one webserver host - amongst others. In general I prefer one virtual host per site, which has the advantage that - from a webservers perspective - you have clearly seperated installations and configurations for each site. Moreover it provides the ability to extend the capabilities of single hosts while narrow down others, depending on your needs. Last not least you can enable or disable single hosts while not affecting the others.
+
+Hence, I would like to show an alternative approach of a multisite Kirby installation, which in my point of view provides several additional advantages.
+
+## Preconditions
+
+The following setup has been tested on Linux OS (Ubuntu 18.04 and Debian 9) together with Apache HTTP Server 2.4.
+
+## In the filesystem
+
+The proposed filesystem layout in (link: docs/cookbook/setup/multisite text: the above mentioned recipe) looks like this:
+
+```filesystem
+  kirby/
+  my-site.com/
+    content/
+    media/
+    site/
+    assets/
+  my-shop.com/
+    content/
+    media/
+    site/
+    assets/
+  index.php
+```
+Looking to it from a higher directory level, it could be seen as follows:
+
+```filesystem
+/srv/www/
+  docroot/
+    kirby/
+    my-site.com/
+      content/
+      media/
+      site/
+      assets/
+    my-shop.com/
+      content/
+      media/
+      site/
+      assets/
+    index.php
+```
+whereby the webservers virtual or real host document root is pointing to `docroot`, which of course could be anywhere in your filesystem.
+
+Let's change that to a setup in which every website is served by a dedicated webserver host and therefore the filesystem would look like the following:
+
+```filesystem
+/srv/www/
+  my-site.com/
+    kirby/
+    content/
+    media/
+    site/
+    assets/
+    index.php
+  my-shop.com/
+    kirby/
+    content/
+    media/
+    site/
+    assets/
+    index.php
+```
+whereupon the fellow Kirby administrator will notice nothing new beside of two Kirby installations side-by-side.
+
+However, we can move the `kirby`folder to a location outside of all document roots and replace it with a symbolic link which points to the moved `kirby` folder:
+
+```filesystem
+/srv/www/
+  kirby/
+  my-site.com/
+    kirby -> /srv/www/kirby
+    content/
+    media/
+    site/
+    assets/
+    index.php
+  my-shop.com/
+    kirby -> /srv/www/kirby
+    content/
+    media/
+    site/
+    assets/
+    index.php
+```
+
+The symbolic links are created like this:
+
+```bash
+user@host:/srv/www/my-site.com $ ln -s /srv/www/kirby kirby
+user@host:/srv/www/my-shop.com $ ln -s /srv/www/kirby kirby
+```
+
+## In `index.php`
+
+php can follow the symlinked `kirby` directory like every else. However, since in `kirby/config/roots.php` the root to `index` is set up like this:
```

Comment:
PHP not php

Reply:
Changed with fb99913a943c31d0efad68b73fc6e73e4d498cce